src/Process/PlanarTileProcessing.js

```diff
@@ -9,47 +8,24 @@ export function planarCulling(node, camera) {
     return !frustumCullingOBB(node, camera);
 }
 
-function computeNodeSSE(camera, node) {
-    var vFOV = camera.FOV * Math.PI / 180;
+function _isTileBigOnScreen(camera, node) {
+    const onScreen = camera.box3DSizeOnScreen(node.geometry.OBB.box3D, node.matrixWorld);
 
-    const tmp1 = new THREE.Vector3();
-    tmp1.setFromMatrixPosition(node.matrixWorld);
-    const tmp2 = new THREE.Vector3();
-    tmp2.setFromMatrixPosition(camera.camera3D.matrixWorld);
-    var diff = tmp2.sub(tmp1);
-    const dim = node.extent.dimensions();
-
-    var d = Math.max(0.1, diff.length() - (new THREE.Vector3(dim.x, dim.y, dim.z)).length() * 0.5);
-    var height = 2 * Math.tan(vFOV / 2) * d;
-
-    var dot = diff.normalize().z;
-
-    var ratio = (dim.x * dot) / height;
+    const dim = {
+        x: 0.5 * (onScreen.max.x - onScreen.min.x),
+        y: 0.5 * (onScreen.max.y - onScreen.min.y),
+    };
 
-    if (ratio >= 0.25) {
-        return 7;
-    }
-    return 1;
+    return (dim.x > 0.3 && dim.y >= 0.3);
```

Comment:
`0.5` is needed because `box3DSizeOnScreen` result goes from `-1` to `+1` (for each axis), so we get back to `[0, 1]`.

The `>` vs `>=` is a typo, I'll fix this.

The `0.3` value are `threshold` that looks good on the sole example of planar mode that we have.
I can make them configurable if you want but honestly this PR is a first iteration and we will probably need to adjust it later (for instance to depend on the resolution of the images we get from the layer).